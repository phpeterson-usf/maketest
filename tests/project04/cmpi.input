-h $(repo_dir)/cmpi.hex $(TESTS_DIR)/cmpi.s
