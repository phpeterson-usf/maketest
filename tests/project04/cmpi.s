main:
    cmp r1, #7
    bx lr
