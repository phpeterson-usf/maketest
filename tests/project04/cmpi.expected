v2.0 raw
e3510007
e12fff1e
